utils/build-script-impl

```diff
@@ -1281,6 +1281,36 @@ function cmake_config_opt() {
     fi
 }
 
+function copy_embedded_compiler_rt_builtins_from_darwin_host_toolchain() {
+    local clang_dest_dir=$1
+
+    HOST_CXX_DIR=$(dirname "${HOST_CXX}")
+    HOST_LIB_CLANG_DIR="${HOST_CXX_DIR}/../lib/clang"
+    DEST_LIB_CLANG_DIR="${clang_dest_dir}/lib/clang"
+
+    if [[ -d "${HOST_LIB_CLANG_DIR}" ]] && [[ -d "${DEST_LIB_CLANG_DIR}" ]]; then
+        DEST_CXX_BUILTINS_VERSION=$(ls "${DEST_LIB_CLANG_DIR}" | awk '{print $0}')
```

Comment:
Is there some guarantee that only a single file will be present?  Seems that it would be nicer to use `ls -1` though instead of the `awk`.

Reply:
That should be the case because we only build a single version of LLVM, and in my (little) experience I did not see more than one folder at that path.

Simplified invocation as suggested as part of 
0ca06c2